wcs-functions.php

```diff
@@ -134,77 +139,53 @@ function wcs_create_subscription( $args = array() ) {
 		return new WP_Error( 'woocommerce_subscription_invalid_start_date_format', _x( 'Invalid date. The date must be a string and of the format: "Y-m-d H:i:s".', 'Error message while creating a subscription', 'woocommerce-subscriptions' ) );
 	}
 
-	// check customer id is set
+	// Check customer id is set.
 	if ( empty( $args['customer_id'] ) || ! is_numeric( $args['customer_id'] ) || $args['customer_id'] <= 0 ) {
 		return new WP_Error( 'woocommerce_subscription_invalid_customer_id', _x( 'Invalid subscription customer_id.', 'Error message while creating a subscription', 'woocommerce-subscriptions' ) );
 	}
 
-	// check the billing period
+	// Check the billing period.
 	if ( empty( $args['billing_period'] ) || ! in_array( strtolower( $args['billing_period'] ), array_keys( wcs_get_subscription_period_strings() ) ) ) {
```

Comment:
```suggestion
	if ( empty( $args['billing_period'] ) || ! in_array( strtolower( $args['billing_period'] ), array_keys( wcs_get_subscription_period_strings() ), true ) ) {
```
PHPCS warning fix: Add third argument (`$strict = true`) to `in_array` to fix `StrictInArray.MissingTrueStrict` (Not using strict comparison for in_array; supply true for third argument).

Reply:
Fixed in ae559089 using the even more awesome `array_key_exists` function